RELEASE.md

```diff
@@ -0,0 +1,36 @@
+# Making a new release of AeoLiS
+
+We publish the package on PyPI. Only administrators can create new releases.
+
+## 1. Prepare Release
+ All of the Python
+packaging instructions are in the the following files:
+
+* `aoelis/VERSION`
+* `setup.cfg`
+* `setup.py`
+
+A. Before generating a package, we first need to install `build`.
+
+```bash
+pip install build 
+```
+
```

Comment:
Include a step here to create a branch `release/x.y.z` from `main` to add the changes for the preparation for the release. This would help keep the documentation consistent as step E mentions to merge the changes to `main`.